File: src/app/types/prediction/Bet.ts
export enum Bet {
  OVER_ZERO_FIVE,
  OVER_ONE_FIVE,
}

export enum BetType {
  OVER,
  UNDER,
}

export interface BetTypeDefinition {
  betType: BetType;
  bets: Bet[];
}

export const OVER: BetTypeDefinition = {
  betType: BetType.OVER,
  bets: [Bet.OVER_ZERO_FIVE, Bet.OVER_ONE_FIVE],
};

File: footystats-tools-backend/src/main/java/de/footystats/tools/services/prediction/Bet.java
package de.footystats.tools.services.prediction;

import java.util.EnumSet;

public enum Bet {
    OVER_ZERO_FIVE,
    OVER_ONE_FIVE,
    BTTS_YES,
    BTTS_NO,
    ;

	/**
	 * Method used to control active bets. Active bets are bets we do predictions
	 * and bet prediction quality for.
	 *
	 * @return s. description.
	 */
	public static EnumSet<Bet> activeBets(){
		return EnumSet.of(OVER_ZERO_FIVE, OVER_ONE_FIVE, BTTS_YES);
	}
}
